omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_8/FormSearchHandler1_8.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.search.openmrs1_8;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.openmrs.Form;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.RequestContext;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.resource.api.PageableResult;
+import org.openmrs.module.webservices.rest.web.resource.api.SearchConfig;
+import org.openmrs.module.webservices.rest.web.resource.api.SearchHandler;
+import org.openmrs.module.webservices.rest.web.resource.api.SearchQuery;
+import org.openmrs.module.webservices.rest.web.resource.impl.EmptySearchResult;
+import org.openmrs.module.webservices.rest.web.resource.impl.NeedsPaging;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
+import org.springframework.stereotype.Component;
+
+@Component
+public class FormSearchHandler1_8 implements SearchHandler {
+
+	private final SearchConfig searchConfig = new SearchConfig("default", RestConstants.VERSION_1 + "/form", Collections.singletonList("1.8 - 2.*"),
+
+			new SearchQuery.Builder(
+					"Allows you to find form by published and unpublished status")
```

Comment:
The above sounds as if two parameters are used. One for published and the other for unpublished.

Reply:
@dkayiwa  Basically one passes one parameter which is published with a value either true or false, But I could only limit it to published

Reply:
@abertnamanya Daniel is referring to the message, it's unclear(suggests two parameters published and unpublished), consider changing to a more clear message, something like `Allows you to filter forms by published status`